Tella/Data/Crypto/CryptoManager.swift

```diff
@@ -316,14 +323,62 @@ extension CryptoManager: CryptoManagerInterface {
     }
     
     func decrypt(_ data: Data) -> Data? {
-
+        
         guard let metaPrivateKey = self.metaPrivateKey else {
             debugLog("Failed to recover private key", space: .crypto)
             return nil
         }
         return decrypt(data, metaPrivateKey)
     }
     
+    func encryptFilePath(inputFileURL: URL, outputFileURL: URL) -> Bool {
```

Comment:
let's add comments or documentation to explain the purpose of the function, parameters, and any potential side effects?